src/main/kotlin/no/nav/syfo/melding/kafka/legeerklaring/LegeerklaringDTO.kt

```diff
@@ -151,25 +151,25 @@ data class Signatur(
     val tlfNummer: String?,
 )
 
-fun LegeerklaringDTO.toMeldingFraBehandler() =
+fun LegeerklaringDTO.toMeldingFraBehandler(parentRef: UUID?) =
```

Comment:
Mulig eg misforstår noe her, men ut ifra bruken så ser det ikke ut som `parentRef` kan være `null` her 🤔
```suggestion
fun LegeerklaringDTO.toMeldingFraBehandler(parentRef: UUID) =
```